.scalafix.conf

```diff
@@ -0,0 +1,9 @@
+
+rules = [
+  ExplicitResultTypes,
+  RemoveUnused,
+  DisableSyntax,
+  LeakingImplicitClassVal,
+  NoValInForComprehension,
+  ProcedureSyntax
+]
```

Comment:
It is not clear to me why this file is added by the PR. It seems to be already here in the repository :thinking:

Reply:
I must have some mistake during the rebase :(

Reply:
May I ask you to fix the Git history, and to possibly squash your commits? This will make the review process easier.